src/pages/WalletPage.vue

```diff
@@ -649,11 +649,11 @@ export default {
 
     // get token to receive tokens from a link
     if (params.get("token") || hash.includes("token")) {
-      let tokenBase64 = params.get("token") || hash.split("token=")[1];
+      const tokenBase64 = params.get("token") || hash.split("token=")[1];
       // make sure to react only to tokens not in the users history
       let seen = false;
-      for (var i = 0; i < this.historyTokens.length; i++) {
-        var thisToken = this.historyTokens[i].token;
+      for (const i = 0; i < this.historyTokens.length; i++) {
```

Comment:
```suggestion
      for (let i = 0; i < this.historyTokens.length; i++) {
```

Reply:
> Hm, I wonder why this was not caught by the build. Is there a way how to check this during build time and not run time?

Found it, I need to add `"no-const-assign": "error",` rule to eslintrc - added to 4f862d1ca6af03db347c29b07b1f2e25dec1ccd0